contracts/core/Allo.sol

```diff
@@ -243,24 +243,32 @@ contract Allo is
         _updateBaseFee(_baseFee);
     }
 
-    /// @notice Add a pool manager
+    /// @notice Add multiple pool managers
     /// @dev Emits 'RoleGranted()' event. 'msg.sender' must be a pool admin.
     /// @param _poolId ID of the pool
-    /// @param _manager The address to add
-    function addPoolManager(uint256 _poolId, address _manager) external onlyPoolAdmin(_poolId) {
-        // Reverts if the address is the zero address with 'ZERO_ADDRESS()'
-        if (_manager == address(0)) revert ZERO_ADDRESS();
+    /// @param _managers The addresses to add
+    function addPoolManagers(uint256 _poolId, address[] calldata _managers) external onlyPoolAdmin(_poolId) {
+        for (uint256 i; i < _managers.length;) {
+            _addPoolManager(_poolId, _managers[i]);
```

Comment:
Do we need the internal function ? All we have is a zero check which is an extra check than what is on removePoolManagers
Might be worth keeping it consistent with removePoolManagers.

<img width="891" alt="image" src="https://github.com/allo-protocol/allo-v2/assets/5358146/e835dbd6-829e-4db1-96b4-72333aff83e9">

Reply:
The internal is made to avoid repeating code ([used in _createPool](https://github.com/allo-protocol/allo-v2/blob/feat/multiple-managers/contracts/core/Allo.sol#L459)). And `removePoolManagers` does not have an internal because is only one line. I can create `_removePoolManager` for consistency if needed.